cuegui/cuegui/FrameMonitorTree.py

```diff
@@ -877,6 +880,8 @@ def __init__(self, widget, filterSelectedLayersCallback):
             self.__menuActions.frames().addAction(self, "xdiff2")
 
         self.__menuActions.frames().addAction(self, "useLocalCores")
+        if bool(int(QtGui.qApp.settings.value("AllowDeeding", 0))):
```

Comment:
Should the line above this one be removed? Otherwise it seems like, if this condition is true, we're adding the same action twice.